test/api/v3/integration/chat/POST-chat.test.js

```diff
@@ -155,4 +161,28 @@ describe('POST /chat', () => {
     expect(message.message.id).to.exist;
     expect(memberWithNotification.newMessages[`${group._id}`]).to.exist;
   });
+
+  it('Returns an error when the user has been posting too many messages', async () => {
```

Comment:
I don't think you need to mock a timer. You can just hard code the timestamp. I think this is better covered in a unit test. The integration test shouldn't need to be so specific.